WcaOnRails/vendor/assets/javascripts/jquery.wca-autocomplete.js

```diff
@@ -15,10 +15,12 @@
       var posts_search = $(that).hasClass("wca-autocomplete-posts_search");
 
       var delimiter = ',';
-      var searchFields = [];
-      searchFields = searchFields.concat([ 'wca_id', 'name' ]); // user search fields
-      searchFields = searchFields.concat([ 'id', 'name', 'cellName', 'cityName', 'countryId' ]); // competition search fields
-      searchFields = searchFields.concat([ 'title', 'body' ]); // post search fields
+      var searchFields = [
+        'wca_id', 'name', // user search fields
+        'id', 'cellName', 'cityName', 'countryId', // competition search fields ('name' already added)
+        'title', 'body', // post search fields
+        'text', // regulation search fields ('id' already added)
+      ];
```

Comment:
Nit: rather than commenting about the omitted fields, could we write them all and call [`_.uniq`](https://lodash.com/docs/4.15.0#uniq) to get rid of them?